client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -42,6 +44,7 @@ const StatusBox: React.FC<IStatusBoxProps> = ({
     <Wrapper icon={null}>
       <Inner>
         <div className="text">
+          <H3>{auditName}</H3>
```

Comment:
It looks a bit weird to me to have both the audit name and the headline be styled as headers. I'm not totally sure what a better option would be, but they seem to visually compete.

If we do really want them to both be headers, I think a little more spacing in between them would be helpful.

Reply:
@jonahkagan Yeah, I agree. I tried putting in a `<hr />` between them and it definitely creates a clear distinction. Let me know what you think. @mcchilders if you could weigh in here with design ideas that'd be awesome too.

Reply:
The hr definitely creates a distinction, but I worry that it takes up a lot of vertical screen real estate. I have been playing around a bit but haven't come up with much except for adding a little bit more spacing between the two headers.

Reply:
Good point. I've reduced the spacing around the hr, and I think it looks better. I like having the clear distinction better than just spacing between the headers myself.

Reply:
Hmm, but the hr extends all the way to the other end and interferes with the button when there's a button:
<img width="1035" alt="Screen Shot 2021-03-24 at 11 25 51 AM" src="https://user-images.githubusercontent.com/530106/112364467-c948ba80-8c93-11eb-8362-0e3df0a95773.png">

Reply:
Oh darnit, forgot about the button. :thinking: Okay, I made it shorter as well, and I think that should do it.

Reply:
Sorry, responded by email but it apparently didn't put it in here! I don't really like the horizontal line, but otherwise it looks fine to me.